test/parallel_api/iterator/input_data_sweep.h

```diff
@@ -54,6 +54,7 @@ wrap_recurse(Policy&& exec, InputIterator1 first, InputIterator1 last, InputIter
 
     //Run the tests
     auto get_expect = [n](auto exp) {
+        (void)n; // avoid unused-lambda-capture, bug https://bugs.llvm.org/show_bug.cgi?id=35450
```

Comment:
It's unfortunate the the `[[maybe_unused]]` attribute doesn't work in lambda capture lists, but it seems like this is the best that can be done here.

Reply:
Yes, good note. `[[maybe_unused]]` would fit here it it was possible.